.env

```
@@ -330,6 +330,7 @@ VITE_JITO_BLOCK_ENGINE_URL=https://mainnet.block-engine.jito.wtf
 
 # relay
 VITE_RELAY_API_URL=https://api.relay.link
+VITE_RELAY_API_KEY=d62274fb-1edd-481a-b952-632896ebf830
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_RELAY_API_KEY key should go before the VITE_RELAY_API_URL key

(UnorderedKey)